packages/bitcore-node/test/unit/models/wallet.unit.ts

```diff
@@ -1,7 +1,58 @@
-import {expect} from 'chai';
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { WalletModel, IWalletModel } from '../../../src/models/wallet';
+import { WalletAddressModel } from '../../../src/models/walletAddress';
 
-describe('Wallet Model', function(){
-    it('should have a test which runs', function(){
-        expect(true).to.equal(true);
+describe('Wallet Model', function () {
+
+  describe('_apiTransform', () => {
+    it('should return the transform object with wallet info', () => {
```

Comment:
Since `_apiTransform` has two paths from it's options, one where it returns and object and one where it returns an object and one where it returns JSON.  We need to test that both of them work.  So you should make another test to make sure that it returns proper JSON.

Reply:
If the unit tests of `_apiTransform` aren't testing both paths can you do that as well

Reply:
resolved